libs/designsystem/kirby-ionic-module/src/kirby-ionic.module.ts

```diff
@@ -1,15 +1,25 @@
 import { NgModule } from '@angular/core';
-import { IonicModule } from '@ionic/angular';
+import { AnimationController, IonicModule, isPlatform } from '@ionic/angular';
 
+const getIonicConfig = () => {
+  let config: any = {
+    mode: 'ios',
+    inputShims: true,
+    scrollAssist: true,
+    scrollPadding: false,
+  };
+
+  if (!isPlatform('hybrid')) {
```

Comment:
Are we sure we only wanna do this on hybrid? Wouldn't it make sense to get an app-like experience when running in the browser on devices? 🤔

Reply:
Just talked with the requester of this feature, and they do want to have a more web-like behavior on mobile web.